CoreScriptsRoot/Modules/Server/ClientChat/ChatBar.lua

```diff
@@ -317,53 +318,86 @@ function methods:SetChannelTarget(targetChannel)
 end
 
 function methods:FadeOutBackground(duration)
-	self.BackgroundTweener:Tween(duration, 1)
+	self.AnimParams.Background_TargetAlpha = 1
+	--self.BackgroundTweener:Tween(duration, 1)
 	--self:FadeOutText(duration)
 end
 
 function methods:FadeInBackground(duration)
-	self.BackgroundTweener:Tween(duration, 0)
+	self.AnimParams.Background_TargetAlpha = 0.8
+	--self.BackgroundTweener:Tween(duration, 0)
 	--self:FadeInText(duration)
 end
 
 function methods:FadeOutText(duration)
-	self.TextTweener:Tween(duration, 1)
+	self.AnimParams.Text_TargetAlpha = 1
 end
 
 function methods:FadeInText(duration)
-	self.TextTweener:Tween(duration, 0)
+	self.AnimParams.Text_TargetAlpha = 0
 end
 
 function methods:CreateTweeners()
-	self.BackgroundTweener:CancelTween()
-	self.TextTweener:CancelTween()
-
-	self.BackgroundTweener = moduleTransparencyTweener.new()
-	self.TextTweener = moduleTransparencyTweener.new()
-
-	--// Register BackgroundTweener objects and properties
-	self.BackgroundTweener:RegisterTweenObjectProperty(self.GuiObject, "BackgroundTransparency")
-	self.BackgroundTweener:RegisterTweenObjectProperty(self.GuiObjects.TextBoxFrame, "BackgroundTransparency")
+	-- self.BackgroundTweener:CancelTween()
```

Comment:
There is a lot of commented out code in this file, if we are no longer using it we should delete this dead-code. If we ever need this code again we can use revision history.